src/Controls/src/Core/Layout/AbsoluteLayout.cs

```diff
@@ -50,7 +62,21 @@ public static Rect GetLayoutBounds(BindableObject bindable)
 			return (Rect)bindable.GetValue(LayoutBoundsProperty);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/AbsoluteLayout.xml" path="//Member[@MemberName='SetLayoutFlags'][1]/Docs/*" />
+		/// <summary> Sets the layout flags of a view that will be used to interpret the layout bounds set on it when it is added to the layout.</summary>
+		/// <remarks>
+		/// <para>This method supports the <c>AbsoluteLayout.LayoutFlags</c> XAML attached property. In XAML, Application developers can specify the following <see cref = "T:Microsoft.Maui.Layouts.AbsoluteLayoutFlags" /> enumeration value names for the value of this property on the children of a <see cref = "T:Microsoft.Maui.Controls.AbsoluteLayout" />:</para>
+		/// <list type = "bullet">
```

Comment:
Not sure if we want to list all the members of an enum? If the enum changes this is out of date. I'd prefer to link the correct type as mentioned above. That way people can navigate there and see and IntelliSense will help them with the available values otherwise.